### PR TITLE
Matlab

### PR DESCRIPTION
**Pull request by jburel**
capitalize matlab
I fixed a few older events and a post too

### CODE REVIEW

File: omero/downloads/index.html [hunk 1]
```diff
@@ -91,9 +91,9 @@ <h6>omero_ij-{{ site.omero.insight.version }}-all.jar</h6>
                 <div class="card card-download">
                     <div class="card-section">
                         <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_matlab.svg">
-                        <h5>Matlab</h5>
+                        <h5>MATLAB</h5>
```
Comment (sbesson) [inline omero/downloads/index.html:94]: Should I close https://github.com/ome/www.openmicroscopy.org/pull/530 in favor of this PR then?

Comment (jburel) [reply omero/downloads/index.html:94]: sorry I did not realise you had a PR
I will remove my commit since you referenced your PR in the post

Comment (sbesson) [reply omero/downloads/index.html:94]: I am happy to have a single PR combining all the related changes. Maybe cherry-pick my commit and close my PR?

**Comment by jburel** [comment]
closing in favour of #530.
I pushed 172f384 to #530

**Pull request closed without merging**